recipes/selenium/Dockerfile

```diff
@@ -89,7 +87,12 @@ RUN echo "export JAVA_HOME=/opt/jdk$JAVA_VERSION_PREFIX\nexport M2_HOME=/home/us
 
 WORKDIR /projects
 
-CMD /usr/bin/supervisord -c /opt/supervisord.conf & \
+CMD if [ -z ${WEBDRIVER_VERSION+x} ]; then WEBDRIVER_VERSION=$(curl -s http://chromedriver.storage.googleapis.com/LATEST_RELEASE); fi && \
+    echo "WEBDRIVER_VERSION: "${WEBDRIVER_VERSION} && \
+    wget -O /home/user/chromedriver_linux64.zip -q http://chromedriver.storage.googleapis.com/${WEBDRIVER_VERSION}/chromedriver_linux64.zip && \
+    unzip -q /home/user/chromedriver_linux64.zip  -d /home/user && \
+    rm /home/user/chromedriver_linux64.zip && \
+    /usr/bin/supervisord -c /opt/supervisord.conf & \
     cd /home/user && sleep 3 && \
-    ./chromedriver --port=4444 --whitelisted-ips='' & \
+    /home/user/chromedriver --port=4444 --whitelisted-ips='' & \
```

Comment:
The latest version will be used until pass the specific version of the driver.
It is useful for selenium tests not to rebuild image whenever we need a different version of the webdriver.